modules/platforms/cpp/DEVNOTES.txt

```diff
@@ -8,62 +8,6 @@ further instructions.
 For details on ODBC driver installation and usage please refer to
 $IGNITE_HOME/platforms/cpp/odbc/README.txt.
 
-Building on Linux With Autotools
-----------------------------------
-
-Common Requirements:
-
- * GCC, g++, autotools, automake, and libtool must be installed
- * OpenSSL, 1.0 or later
- * Java Development Kit (JDK) must be installed: https://java.com/en/download/index.jsp
- * JAVA_HOME environment variable must be set pointing to Java installation directory.
- * IGNITE_HOME environment variable must be set to Ignite installation directory.
- * To build ODBC driver you need to install ODBC Driver Manager on you system.
-   Apache Ignite has been tested with UnixODBC. Make sure that you install development
-   version of the driver manager which contains header files that are necessary for
-   building a driver.
-
-Building the Apache Ignite C++ components:
-
- * Navigate to the directory $IGNITE_HOME/platforms/cpp
- * Execute the following commands one by one to build the project:
-    * libtoolize && aclocal && autoheader && automake --add-missing && autoreconf
-    * ./configure
-    * make
-
-Among standard included by autotools options configure script also include following
-Ignite-specific options:
- * --enable-odbc - build included ODBC driver. Disabled by default.
- * --enable-core - build Ignite core library. Enabled by default.
- * --enable-thin-client - build thin client library. Enabled by default.
- * --enable-node - build stand-alone node executable. Enabled by default.
-
-If you only need to build ODBC driver and do not want to build anything else you may just
-use configure script with the following arguments:
- ./configure --enable-odbc --disable-core --disable-thin-client --disable-node
-
-With the config like that you should not have any ignite dependencies as jvm.dll installed
-to build ODBC driver.
-
-Similarly if you only want to build only thin client you may use configure script with
-the following arguments:
- ./configure --disable-core --disable-node
-
-NOTE: You may want to use "make install" command to install Ignite libraries and headers
-for your operation system. Note however that this action may require superuser privileges.
-
-Building tests:
-
-Boost framework is required to build tests. The following boost libraries are used:
- * boost_unit_test_framework
- * boost_thread
- * boost_system
- * boost_chrono
-
-Ignite expects for boost libraries and header to be found under default system paths, so
-it is recommended to use package repository for your OS to install boost package for the
-development.
-
 Building on Linux and Mac OS X with CMake
```

Comment:
Should we mention that building with CMake on Windows is not supported yet? Just to avoid unnecessary user list threads